### PR TITLE
Fix sonar issues usage of map entry set

### PR DESCRIPTION
**Pull request by Coduz**
This PR fixes other issue reported by Sonar analisys.

More info at: https://sonar.eclipse.org/drilldown/issues/144827

Rules fixed are on commit message

### CODE REVIEW

File: commons/src/main/java/org/eclipse/kapua/commons/event/ServiceEventScope.java [hunk 1]
```diff
@@ -91,6 +92,10 @@ public static void end() {
             eventContextThdLocal.set(null);
         }
 
-        eventStack.pop();
+        if (eventStack != null) {
+            eventStack.pop();
+        } else {
+            throw KapuaRuntimeException.internalError("Event stack should be 'null'");
```
Comment (riccardomodanese) [inline commons/src/main/java/org/eclipse/kapua/commons/event/ServiceEventScope.java]: shouldn't be 'null' :)

Comment (Coduz) [reply commons/src/main/java/org/eclipse/kapua/commons/event/ServiceEventScope.java]: Fixed 👍

**Comment by Coduz** [comment]
After
screen shot 2018-05-17 at 9 30 36 am

**Comment by Coduz** [comment]
Difference over the day:
screen shot 2018-05-17 at 9 30 36 am

**Pull request merged**